data/operators/amenity/hospital.json

```diff
@@ -26,6 +26,118 @@
     }
   },
   "items": [
+    {
+      "displayName": "KwaZulu-Natal Department of Health",
+      "locationSet": {"include": ["za"]},
+      "matchNames": [
+        "operator/KZN Department of Health",
+        "operator/KZN Health Department",
+        "operator/KwaZulu-Natal Health Department"
+      ],
+      "tags": {
+        "amenity": "hospital",
+        "healthcare": "hospital",
+        "operator": "KwaZulu-Natal Department of Health",
+        "operator:type": "government",
+        "operator:wikidata": "Q116647976"
+      }
+    },
+    {
+      "displayName": "Mpumalanga Deparment of Health",
+      "locationSet": {"include": ["za"]},
+      "matchNames": ["operator/Mpumalanga Health Department"],
+      "tags": {
+        "amenity": "hospital",
+        "healthcare": "hospital",
+        "operator": "Mpumalanga Deparment of Health",
+        "operator:type": "government",
+        "operator:wikidata": "Q116648411"
+      }
+    },
+    {
+      "displayName": "Free State Department of Health",
+      "locationSet": {"include": ["za"]},
+      "matchNames": ["operator/Free State Health Department"],
+      "tags": {
+        "amenity": "hospital",
+        "healthcare": "hospital",
+        "operator": "Q116648781",
+        "operator:type": "government",
+        "operator:wikidata": "Free State Department of Health"
```

Comment:
Is there a wikidata for "_Free State Department of Health_"?

Reply:
Oops, I'll fix that now

Reply:
Done, but should I have removed the id, as it now looks a bit odd?

Reply:
The ID's are auto generated so you don't need to worry about those at all 👍